src/components/Items/items.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Item from "./item/item";
+import { connect } from "react-redux";
+import { mark_checked, removeTodo} from "../../actions/actions";
+
+function Items(props){
+    let unchecked_items = [];
+    let checked_items = [];
+    const {items=[], filtered_items = []} = props;
+    const itemsToRender = filtered_items.length ? filtered_items : items;
+    
+    itemsToRender.forEach(item => {
+        if (item.checked === true)
+            checked_items.push(
+                <Item key={item.id} {...item} onDel={props.onDel} check={props.check}/>
+            );
+        else
+                unchecked_items.push(
+                    <Item key={item.id} {...item} onDel={props.onDel} check={props.check}/>
```

Comment:
Why are you using props.check instead of item.checked?

Reply:
That's a function and not the checked property in items object